carma_cooperative_perception/src/msg_conversion.cpp

```diff
@@ -314,8 +314,8 @@ auto to_detection_msg(
     detection.pose = object.pose;
   }
 
-  if (object.presence_vector & object.VELOCITY_INST_PRESENCE_VECTOR) {
-    detection.twist = object.velocity_inst;
+  if (object.presence_vector & object.VELOCITY_PRESENCE_VECTOR) {
+    detection.twist = object.velocity;
```

Comment:
There seems to be several more instances of velocity_inst being used in this file and its corresponding test file. Do we not need those changed?

Reply:
Yes. Good catch.